packages/core/admin/ee/admin/pages/SettingsPage/pages/ReviewWorkflows/tests/ReviewWorkflows.test.js

```diff
@@ -53,35 +82,20 @@ const setup = (props) => render(<ComponentFixture {...props} />);
 const user = userEvent.setup();
```

Comment:
I'm not sure you should initialise this here https://testing-library.com/docs/user-event/intro#writing-tests-with-userevent

Reply:
Good catch. Done in 0f01bb908bf0f2bac925e84736f5d9c6895320d5